nucliadb/nucliadb/ingest/consumer/metrics.py

```diff
@@ -0,0 +1,25 @@
+# Copyright (C) 2021 Bosutech XXI S.L.
+#
+# nucliadb is offered under the AGPL v3.0 and as commercial software.
+# For commercial licensing, contact us at info@nuclia.com.
+#
+# AGPL:
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+from nucliadb_telemetry import metrics
+
+total_messages = metrics.Counter(
+    "nuclia_subscriber_messages", labels={"type": "", "action": "scheduled"}
+)
+handler_histo = metrics.Observer("nuclia_shard_creator_handler", labels={"type": ""})
```

Comment:
Should the metric name be `nucliadb_handler` and then use `type` label to distinguish between handlers?

Reply:
blah, definitely. I forgot to fix these. Thanks.